projects/jetstack/cert-manager/README.md

```diff
@@ -17,3 +17,22 @@ You can find the latest versions of these images on ECR Public Gallery.
 [cert-manager Controller](https://gallery.ecr.aws/eks-anywhere/jetstack/cert-manager-controller) | 
 [CA injector](https://gallery.ecr.aws/eks-anywhere/jetstack/cert-manager-cainjector) | 
 [cert-manager Webhook Server](https://gallery.ecr.aws/eks-anywhere/jetstack/cert-manager-webhook)
+
+### Updating
+
+1. Update cert-manager tag when updating cluster-api tag if cluster-api is using a newer tag.
+   Use the same tag that cluster-api uses by default. For instance [cluster-api v1.0.1 uses cert-manager v1.5.3 by default]
+   (https://github.com/kubernetes-sigs/cluster-api/blob/v1.0.1/cmd/clusterctl/client/config/cert_manager_client.go#L30) so when updating
+   to cluster-api tag v1.0.1, update cert-manager tag to v1.5.3
+1. Review releases and changelogs in upstream [repo](https://github.com/jetstack/cert-manager) and decide on new version.
+   Please review carefully and if there are questions about changes necessary to eks-anywhere to support the new version
+   and/or automatically update between eks-anywhere version reach out to @jgw or @mrajashree.
+1. Review the patches under patches/ folder and remove any that are either merged upstream or no longer needed.
+1. Update the `GIT_TAG` file to have the new desired version based on the upstream release tags.
+1. Compare the old tag to the new, looking specifically for Makefile changes.
```

Comment:
I think cert-manager uses bazel which can make it hard to really track specific build flags and what not, not sure if were able to pin point any specific places to review?

Reply:
removed this line. Because I think we only need to check what changes were made to cert-manager CRs in the upstream providers and do the same in our custom providers like the etcdadm ones. So I've added instructions for that